src/store.js

```diff
@@ -71,8 +71,11 @@ export default createStore({
         ]
       }
     },
-    clearNotifications (state) {
-      state.notificationsById = {}
+    clearInfoNotificationsById (state, id) {
+      state.notificationsById[id] = state.notificationsById[id].filter(notification => notification.type !== 'info')
+    },
+    clearWarningNotificationsById (state, id) {
+      state.notificationsById[id] = state.notificationsById[id].filter(notification => notification.type !== 'warning')
```

Comment:
why is this now two functions? It might help to add a comment to explain.

Reply:
Added a comment ✅